engine/language-server/src/test/resources/application-test.conf

```diff
@@ -4,6 +4,63 @@ akka.http.server.websocket.periodic-keep-alive-max-idle = 1 second
 akka.loglevel = "ERROR"
 akka.test.timefactor = ${?CI_TEST_TIMEFACTOR}
 akka.test.single-expect-default = 5s
+akka.actor.default-dispatcher = {
+  # Dispatcher is the name of the event-based dispatcher
+  type = Dispatcher
+  # What kind of ExecutionService to use
+  executor = "thread-pool-executor"
+  # Configuration for the thread pool
+  thread-pool-executor {
+    fixed-pool-size = 2
+    # Keep alive time for threads
+    keep-alive-time = 10s
+    # Allow core threads to time out
+    allow-core-timeout = off
+  }
+  # Throughput defines the maximum number of messages to be
+  # processed per actor before the thread jumps to the next actor.
+  # Set to 1 for as fair as possible.
+  throughput = 20
+}
+
+akka.actor.internal-dispatcher = {
+  # Dispatcher is the name of the event-based dispatcher
+  type = Dispatcher
+  # What kind of ExecutionService to use
+  executor = "thread-pool-executor"
+  # Configuration for the thread pool
+  thread-pool-executor {
+    fixed-pool-size = 2
```

Comment:
Have you tried `fixed-pool-size = 1`? Does the application work? If so, _single threaded executor_ is something I tend to prefer...

Reply:
Yes. We are talking about 70s execution time vs 800s on my machine. I think it is not worth it.